test/unit/spec/OAuth2Util_spec.js

```diff
@@ -58,6 +61,42 @@ describe('util/OAuth2Util', function() {
       }).catch(done.fail);
     });
 
+    it.each([
+      'access_denied',
+      'jit_failure_missing_fields',
+      'jit_failure_invalid_login_format',
+      'jit_failure_values_not_match_pattern',
+      'jit_failure_values_too_long',
+      'jit_failure_invalid_locale',
+    ])('shows SDK errors of certain types', function(errCode, done) {
```

Comment:
nit: To easily identify the error code tested, let's include it in the test description:

```suggestion
    ])('shows generic error for errorCode "%s"', function(errCode, done) {
```